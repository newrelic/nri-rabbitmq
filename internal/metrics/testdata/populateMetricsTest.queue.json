[
    {
        "name": "test-name",
        "vhost": "test-vhost",
        "messages_ready_details": {
            "rate": 10.5
        },
        "messages_ready": 243
    }
]