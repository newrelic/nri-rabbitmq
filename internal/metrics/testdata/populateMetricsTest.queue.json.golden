{"clusterName":"testClusterName","displayName":"test-vhost/test-name","entityName":"queue:test-vhost/test-name","event_type":"RabbitmqQueueSample","queue.bindings":2,"queue.messagesReadyDeliveryClients":243,"queue.messagesReadyDeliveryClientsPerSecond":10.5,"reportingEndpoint":":0"}